rc/extra/git-tools.kak

```diff
@@ -97,7 +97,8 @@ Available commands:\n  add\n  rm\n  blame\n  commit\n  checkout\n  diff\n  hide-
     }
 
     update_diff() {
-        git --no-pager diff -U0 "$kak_buffile" 2>/dev/null | perl -e '
+        git rev-parse --show-toplevel >/dev/null 2>&1 || return
+        git --no-pager diff -U0 "$kak_buffile" | perl -e '
```

Comment:
@lenormf what are your thoughts on this approach? I can put the check in my hook, but maybe having it here is better to benefit everyone. Basically the update_diff command will now do nothing if we are not in git repo.

Reply:
Actually, I realized that it defeats the purpose, your point was that people need to see the error, and in this case people will still see nothing in `*debug*`. I'll update.